src/pylorax/monitor.py

```diff
@@ -168,7 +191,7 @@ class LogMonitor(object):
     This needs to be running before the virt-install runs, it expects
     there to be a listener on the port used for the virtio log port.
     """
-    def __init__(self, log_path=None, host="localhost", port=0, timeout=None):
+    def __init__(self, log_path=None, host="localhost", port=0, timeout=None, log_request_handler_class=None):
```

Comment:
You can use `LogRequestHandler` instead of None.

Reply:
Good point! Changed.